go/private/rules/binary.bzl

```diff
@@ -94,6 +95,24 @@ def new_cc_import(
         ),
     )
 
+def _go_cc_aspect_impl(target, _ctx):
```

Comment:
Why `_ctx`? It's not unused.

Reply:
oh it was leftover from when i was using the GoSource, i'll fix